frontend/src/components/product/components/Pagination.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ButtonBase } from "@material-ui/core";
+
+const Pagination = (props: any) => {
+  const { productPerPage, totalPage, paginate } = props;
+
+  const pageNumber = [];
+
+  for (let i = 1; i <= Math.ceil(totalPage / productPerPage); i++) {
+    pageNumber.push(i);
+  }
+
+  return (
+    <nav>
+      {pageNumber.map((number) => (
+        <ButtonBase
+          className="page-link"
+          onClick={() => paginate(number)}
+          style={{ marginLeft: 20, borderWidth: 1, borderColor: "black" }}
+        >
+          {number}
+        </ButtonBase>
+      ))}
```

Comment:
I think we should show button in limit page.
for example, if we are in page 5 and it have 10 total page,
It should display like below.

1 ... 3 4 5 6 7 ... 10

It should display first and last with nearby page that we are now.
(also it should be good, if it have parameter to set nearby range)